src/test/kotlin/no/nav/familie/ef/sak/beregning/OmregningServiceTest.kt

```diff
@@ -102,6 +104,43 @@ internal class OmregningServiceTest : OppslagSpringRunnerTest() {
         clearMocks(iverksettClient, answers = false)
     }
 
+    val fagsakId = UUID.fromString("3549f9e2-ddd1-467d-82be-bfdb6c7f07e1")
+    val behandlingId = UUID.fromString("39c7dc82-adc1-43db-a6f9-64b8e4352ff6")
+
+    /**
+     * Denne brekker hver gang det kommer nytt G-beløp.
+     */
+    @Test
+    fun `Verifiser riktig beløp og intekstjustering`() {
+        val (inntekt, totalinntekt) = lagInntekt(201, 2002, 200003, år)
+        lagSøknadOgVilkårOgVedtak(behandlingId, fagsakId, inntekt, stønadsår = år)
+        val tilkjentYtelse = lagreTilkjentYtelse(behandlingId, stønadsår = år)
+        val iverksettDtoSlot = slot<IverksettOvergangsstønadDto>()
+        // Gitt assert: - skal splittes til to med ny g
+        assertThat(tilkjentYtelse.andelerTilkjentYtelse).hasSize(1)
+        assertThat(totalinntekt.toInt()).isEqualTo(276287)
+
+        // When
+        omregningService.utførGOmregning(fagsakId)
+        verify { iverksettClient.iverksettUtenBrev(capture(iverksettDtoSlot)) }
+        val iverksettDto = iverksettDtoSlot.captured
+
+        // Then
+        assertThat(iverksettDto.vedtak.tilkjentYtelse?.andelerTilkjentYtelse?.size).isEqualTo(2) // skal være splittet
+
+        // Sjekk andel etter ny g omregningsdato
+        val andelTilkjentYtelseOmregnet = finnAndelEtterNyGDato(iverksettDto)!!
```

Comment:
Hadde nesten kunnet brukt last på andeler her, og verifisert at datoet blir satt opp riktig også, samtidig er det vel kanskje noe som allerede er dekket av andre tester? (for beregningsutil eks)

Reply:
jepp - tenker holde denne testen "så enkel som mulig". Jeg tenker vi teser nok her allerede.  Kan godt legge til flere tester som sjekker andre ting?